dashboard/app/controllers/levels_controller.rb

```diff
@@ -203,6 +204,10 @@ def embed_level
     render 'levels/show'
   end
 
+  def set_applab_user_id
+    @applab_user_id = applab_user_id
```

Comment:
Why do you need to set this `@instance` variable vs. just calling this method directly here: https://github.com/code-dot-org/code-dot-org/blob/6d6441dea3458dc14aa272744dc42309c3ab2fff/dashboard/app/helpers/levels_helper.rb#L306